generate_constants.jl

```diff
@@ -0,0 +1,34 @@
+using SciPy.PyCall
```

Comment:
Technically we also need 

```julia
using SciPy
```

here because 

```
@pyinclude(joinpath(pkgdir(SciPy), "src", "scipy_api_list.py"))
```

depends `SciPy`.

Reply:
I found [generate_constants.jl](https://github.com/AtsushiSakai/SciPy.jl/pull/67/files/eae66e6912209d7a4166b4ba7d32dbc0c7244c2d#diff-23e2a7618268cec1d4cd425680a7d76282ae8ac0c7ce85be10f6b1fd866858f7) is just a script to generate `src/scipy_constants.jl` automatically. We don't have to fix it right now.